test/integration_test.tsx

```diff
@@ -73,6 +74,22 @@ function addStaticTests(render: (el: ReactElement) => Wrapper) {
     assert.equal(wrapper.html(), '<button>Click me</button>');
   });
 
+  it('can return HTML content of mixed typed children', () => {
+    function Button({ label }: any) {
+      return (
+        <button>
+          {[null, undefined, true, false, 0n, ' ', label, ' ', 0]}
+        </button>
+      );
+    }
+
+    const wrapper = render(<Button label="Click me" />);
+    assert.equal(
+      wrapper.html(),
+      isStringRenderer ? '0 Click me 0' : '<button>0 Click me 0</button>'
```

Comment:
Can you remind me why the outer `<button>` is missing when using the string renderer?

Reply:
Yea, this confused me at first too. It's cuz the [enzyme string renderer returns a Cheerio root](https://github.com/enzymejs/enzyme/blob/9bb92eb6dad19152766549bb22039264982e673b/packages/enzyme/src/render.js#L22), not an enzyme wrapper.

So the `wrapper` return value represents the `<button>` element and [calling `html` on it returns the HTML of its children](https://github.com/cheeriojs/cheerio/blob/94fc9b1dcc21e747770caf5ea32f005e195acd61/src/api/manipulation.ts#L953). [Calling `wrapper.toString()` returns the full HTML](https://github.com/cheeriojs/cheerio/blob/94fc9b1dcc21e747770caf5ea32f005e195acd61/src/api/manipulation.ts#L977) so you'd get `<button>0 Click me 0</button>`.